contracts/eosio.token/include/eosio.token/eosio.token.hpp

```diff
@@ -104,15 +123,25 @@ namespace eosio {
          static asset get_supply( const name& token_contract_account, const symbol_code& sym_code )
          {
             stats statstable( token_contract_account, sym_code.raw() );
-            const auto& st = statstable.get( sym_code.raw(), "invalid supply symbol code" );
-            return st.supply;
+            return statstable.get( sym_code.raw(), "invalid supply symbol code" ).supply;
+         }
+
+         static asset get_max_supply( const name& token_contract_account, const symbol_code& sym_code )
```

Comment:
What is the naming convention with underscore,  `setmaxsupply` vs `get_max_supply`

Reply:
`setmaxsupply` is a public action and must adhere to the `name` type. 

`get_max_supply` is a static method that doesn't need to adhere to it.